pkgs/top-level/python-packages.nix

```diff
@@ -6951,9 +6951,11 @@ self: super: with self; {
 
   mypy-boto3-builder = callPackage ../development/python-modules/mypy-boto3-builder { };
 
-  mypy-boto3-ebs = callPackage ../development/python-modules/mypy-boto3-ebs { };
-
-  mypy-boto3-s3 = callPackage ../development/python-modules/mypy-boto3-s3 { };
+  inherit (callPackage ../development/python-modules/mypy-boto3 { })
```

Comment:
```suggestion
  inherit (callPackages ../development/python-modules/mypy-boto3 { })
```

Reply:
It seems that `callPackages` is not imported in this scope, do you think it would be better if we would have it here, or `callPackage` will do?